src/components/Modal.js

```diff
@@ -0,0 +1,20 @@
+import "./Modal.css";
+import React from "react";
+
+export default function Modal({ handleClose, show, children }) {
+    const showHideModalClassName = show ? "modal display-block" : "modal display-none";
```

Comment:
You can hide the modal by returning `null`

```
return !show ? null : (<div> ... </div>);
```

Reply:
done